README.md

```diff
@@ -35,13 +35,21 @@ kafka.brokers = my_test_kafka:9092
 EOF
 ```
 
+### Automagical topic name generation
+
+If you set `default_topic_prefix` in `~/.kafka-partition-availability-benchmark.properties`, the
```

Comment:
I thought you made this the default; so if they "set default_topic_prefix" it'll use what they set